casper/src/main/resources/AuthKey.rho

```diff
@@ -68,9 +69,8 @@ in {
   } |
 
   rs!(
-    "04f4b4417f930e6fab5765ac0defcf9fce169982acfd046e7c27f9b14c0804014623c0439e5c8035e9607599a549303b5b6b90cd9685e6965278bddca65dac7510".hexToBytes(),
     (9223372036854775807, bundle+{*AuthKey}),
-    "304402205d2dc163a40804a445237ef6bd181249a68598d146e66d0c96484c6cfeaaaffd02205e537f1e7bae82b99abbdc25c8472c44f5848085a3a3f23357507c05524be4a7".hexToBytes(),
+    *deployerId,
```

Comment:
Nice improvement!

---

Additional task can be to simplify deployers with only one private key for all blessed contract. I'm not sure why are keys created for each contract?!
https://github.com/rchain/rchain/blob/643fd5f3e277cd29d069c263002526c52b3ad248/casper/src/main/scala/coop/rchain/casper/genesis/contracts/StandardDeploys.scala#L32-L55

Reply:
What's your idea with that? Could this be another pr? I want to simplify changes so that I don't have to deal with heavy conflict in https://github.com/rchain/rchain/pull/3485

Reply:
NVM, we need separate keys for blessed deploys.